docs/rgw_setup.md

```diff
@@ -0,0 +1,136 @@
+# Setting up Ceph Object Gateway for different RPC Use Cases
+
+Ceph Object Gateway(rgw) is an object storage interface built on top of librados to provide applications with a RESTful gateway to Ceph Storage Clusters. Ceph Object Storage supports two interfaces  S3 and OpenStack Swift.   
+
+Complete documentation for the Ceph Object Gateway can be found (here.)[http://docs.ceph.com/docs/master/radosgw/]
+
+## Role of rpc-ceph
+The purpose of (rpc-ceph)[https://github.com/rcbops/rpc-ceph] is to provide a consistant versioned set of processes, variables and versioned software for RPC's use.  
+
+### List of Use Cases
+
+* General Installation using ceph-ansible
+* Configuring rgw to use a Stand alone instance of OpenStack Keystone
+* Configuring rgw to use KeyStone and Interact with an OpenStack environment in place of OpenStack Swift
+
+
+
+## Installation of rgw only
+To Install rgw **ONLY** and **NOT** use any of rpc-ceph configured varibles 
+
+### To install rgw addtions to the inventory file.   
+
+Add the **rgw** group with a list of hosts into the ceph environment's inventory file. 
+For example:
+```bash
+[rgw]
+ceph-rgw01
+ceph-rgw02
+ceph-rgw03
+```
+
+### Installation variables
+Set aside the rpc-ceph preconfigured variables
+```bash
+cp playbooks/group_vars/rgws.yml playbooks/group_vars/rgw-rpc-ceph.yml.sample
+cp playbooks/group_vars/rgws.yml.sample playbooks/group_vars/rgws.yml
+```
+**Special Note**  Using this approach will install rgw with the ceph-ansible defaults and you will be responsible for assigning all configuration variables needed for your environment that differ from the ceph-ansible defaults
+
+### Run the deployment
+```bash
+ceph-ansible -i <link to your inventory file> playbooks/deploy-ceph.yml -e@<link to your vars file>
+```
+
+
+
+## Installation of rgw with Stand Alone Openstack KeyStone
+To Install rgw and connect it to a Stand Alone instance of OpenStack Keystone.   Please refer to other documentation for the RPC guidelines for installing and setting up OpenStack Keystone.   This ceph rgw guide assumes you have an working OpenStack Keystone system in place
+
+### To install rgw addtions to the inventory file.   
+
+Add the **rgw** group with a list of hosts into the ceph environment's inventory file. 
+For example:
+```bash
+[rgw]
+ceph-rgw01
+ceph-rgw02
+ceph-rgw03
+
+[keystone_all]
+keystone_host
+```
+
+### Preset variables
+rpc-ceph has already assigned some ceph-ansible variable in the file **(playbooks/group_vars/rgws.yml)[https://raw.githubusercontent.com/rcbops/rpc-ceph/master/playbooks/group_vars/rgws.yml]**  If special needs are required for a specific customer's requirements these variable can be changed or overridden
+
+Make changes these lines from the file **playbooks/group_vars/rgws.yml** because they are used for integration with OpenStack
+```
+# Replace these 
+radosgw_keystone_service_name: "swift"
```

Comment:
I think lets make this a patch to do this rather than asking them to manually do it.
If we change those group_vars/rgws.yml files to those values it seems to just make sense. Then we can remove this step.

Reply:
I don't know how changing them in an OpenStack deploy will affect other things.

Reply:
We can test that, but description one at a minimum should make no difference. I'll put a PR in for this now, and if it passes our keystone-rgw tests it should be fine. Since no openstack services use the RadosGW (glance for images uses direct rbd access, as do cinder/nova) so it would only be swift client (which we test in our keystone_rgw gate).

Reply:
removed reference to changing service name